docs/Accessibility.md

```diff
@@ -23,32 +23,50 @@ If you want to allow trips and stops of unknown wheelchair-accessibility then ad
       },
-      "stops": {
+      "stop": {
         "onlyConsiderAccessible": false,
         "unknownCost": 600,
         "inaccessibleCost": 3600
-      }
+      },
+      "inaccessibleStreetReluctance": 25,
+      "maxSlope": 0.08333,
+      "slopeExceededReluctance": 50,
+      "stairsReluctance": 25
     }
   },
   "updaters": []
 }
 ```
 
-The parameters for `stops` and `trips` mean the following:
+The parameters for `stop`, `trip` and `elevator` mean the following:
+
+| name                     |                                                                                                | default |
+|--------------------------|------------------------------------------------------------------------------------------------|---------|
+| `onlyConsiderAccessible` | Whether to include unknown accessibility and inaccessible stops/trips/elevators in the search. | `false` |
+| `unknownCost`            | The cost to add if an entity has unknown wheelchair accessibility                              | 600     |
+| `inaccessibleCost`       | The cost to add if an entity is known to be inaccessible                                       | 3600    |
 
-| name                     |                                                                                      | default |
-|--------------------------|--------------------------------------------------------------------------------------|---------|
-| `onlyConsiderAccessible` | Whether to include unknown accessibility and inaccessible stops/trips in the search. | `false` |
-| `unknownCost`            | The cost to add if a stop/trip has unknown wheelchair accessibility                  | 600     |
-| `inaccessibleCost`       | The cost to add if a stop/trip is known to be inaccessible                           | 3600    |
+**Note**: Unless your accessibility data coverage is complete you will receive much better results
+by setting `onlyConsiderAccessible=false`, because otherwise you receive barely any results.
 
-**Note**: Unless your accessibility data coverage is complete you will receive much better results by setting `onlyConsiderAccessible=true`, because
-otherwise you receive barely any results.
+Other parameters are:
+
+- `inaccessibleStreetReluctance`: if a street is marked as wheelchair-inaccessible this is the
+  penalty that is applied for wheelchair users. This should be quite high so that those are only
+  chosen as a very last resort. default: 25
+- `maxSlope`: the maximum slope that a wheelchair user can use without incurring routing penalties (
+  leading to those ways being avoided). default: 0.083 (8.3 %)
+- `slopeExceededReluctance`: the multiplier applied to how much you exceed the `maxSlope` and then
+  again multiplied with the regular cost of the street. In other words: how steep should the cost
+  increase when you exceed the maximum slope. Since the percentages are expressed as fractions of 1 
+  (1% is `0.01`) you need a relatively high number. default: 50 
```

Comment:
The other description for this parameter says that if the value is negative, routing on too slopy streets is disabled. Should that also be mentioned here?

Reply:
I've updated the documentation in 79c20466645acea3303c7a2c97c1a43b546c0d94 and added the information about values less than zero.